cmd/file_lint.go

```diff
@@ -220,6 +247,8 @@ func newLintCmd() *cobra.Command {
 	lintCmd.Flags().BoolVarP(&cmdLintOnlyFailures,
 		"display-only-failures", "D", false,
 		"only output results equal to or greater than --fail-severity")
+	lintCmd.Flags().StringVarP(&cmdCustomFunctionsPath, "functions", "f", "",
```

Comment:
I would prefer a descriptive argument name. `--custom-function-path` and then a short option, possibly `-f` for this command.